app/views/case_court_reports/index.html.erb

```diff
@@ -13,19 +13,26 @@
       <div class="field form-group">
         <%= label_tag :case_number, "Case Number:" %>
         <%= select_tag :case_number,
-                      options_from_collection_for_select(@assigned_cases, :case_number, :case_number, 0),
+                      options_from_collection_for_select(
+                        @assigned_cases,
+                        :case_number,
+                        :case_number,
+                        { disabled: @assigned_cases.reject(&:transition_aged_youth).map(&:case_number) }
```

Comment:
Disable ability to choose the non-transitioned cases, as it might use a different template. It will be removed when court report for non-transition aged youth is implemented.

Reply:
I think there can be an indicator to display if the case is transition or non-transition here, so volunteers know which report type they are getting.

Reply:
Sounds good :)